### PR TITLE
Sort media by survey point name

### PR DESCRIPTION
**Pull request by tleraj**
The purpose of this PR is to sort the media carousels on survey details page by point of interest name.

### CODE REVIEW

File: packages/website/src/helpers/surveyMedia.ts [hunk 1]
```diff
@@ -25,8 +25,9 @@ export const getNumberOfSurveyPoints = (surveyMedia: SurveyMedia[]) => {
 };
 
 export const getSurveyPointsByName = (surveyMedia: SurveyMedia[]) => {
+  const sortedByPoiName = sortBy(surveyMedia, (media) => media.poiId?.name);
```
Comment (ericboucher) [inline packages/website/src/helpers/surveyMedia.ts:28]: would that put "other" at the end?

File: packages/website/src/helpers/surveyMedia.ts [hunk 2]
```diff
@@ -25,15 +25,27 @@ export const getNumberOfSurveyPoints = (surveyMedia: SurveyMedia[]) => {
...
-    name,
-    surveyMedia: pointSurveyMedia,
-  }));
+  const groupedMediaArray = Object.entries(groupedMedia).map(
+    ([name, pointSurveyMedia]) => ({
+      name,
+      surveyMedia: pointSurveyMedia,
+    })
+  );
+  const poiNames = Object.keys(groupedMedia);
```
Comment (ericboucher) [inline packages/website/src/helpers/surveyMedia.ts:42]: this seems overly complicated to me. see my other comment

File: packages/website/src/helpers/surveyMedia.ts (hunk 1 above)
Comment (ericboucher) [reply packages/website/src/helpers/surveyMedia.ts:28]: why not just make sure HERE that the null / undefined case is sorted at the end.

Not ideal, but for example doing: => `media.poiId?.name || "zzzz"`

Comment (tleraj) [reply packages/website/src/helpers/surveyMedia.ts:28]: Turns out that `lodash`'s `sortBy` places `null` or `undefined` values at the end. So it is guaranteed that `Other` will be at the end.

**Review by ericboucher — APPROVED** [review]

**Pull request merged**